CHANGELOG

```diff
@@ -1,3 +1,6 @@
+* Tue Apr 11 2017 Nick Markowski <nmarkowski@keywcorp.com> - 7.0.3-0
+- rsyslog::server::enable_selinux now optional, for robustness
```

Comment:
To expand on what rubustness means, the class will no longer fail if the fact isn't found.